sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-batch-scoring/image-classification-multiclass-batch-scoring-non-mlflow-model.ipynb

```diff
@@ -0,0 +1,1202 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1. Connect to Azure Machine Learning Workspace\n",
+    "\n",
+    "The [workspace](https://docs.microsoft.com/en-us/azure/machine-learning/concept-workspace) is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. In this section we will connect to the workspace in which the job will be run.\n",
+    "\n",
+    "## 1.1. Import the required libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "gather": {
+     "logged": 1634852261599
+    },
+    "name": "automl-import"
+   },
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
+    "from azure.identity import DefaultAzureCredential\n",
+    "from azure.ai.ml import MLClient\n",
+    "\n",
+    "from azure.ai.ml.constants import AssetTypes\n",
+    "from azure.ai.ml import Input\n",
+    "from azure.ai.ml.automl import ImageClassificationSearchSpace\n",
+    "from azure.ai.ml.sweep import (\n",
+    "    Choice,\n",
+    "    Uniform,\n",
+    "    BanditPolicy,\n",
+    ")\n",
+    "\n",
+    "from azure.ai.ml import automl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.2. Configure workspace details and get a handle to the workspace\n",
+    "\n",
+    "To connect to a workspace, we need identifier parameters - a subscription, resource group and workspace name. We will use these details in the `MLClient` from `azure.ai.ml` to get a handle to the required Azure Machine Learning workspace. We use the default [default azure authentication](https://docs.microsoft.com/en-us/python/api/azure-identity/azure.identity.defaultazurecredential?view=azure-python) for this tutorial. Check the [configuration notebook](../../configuration.ipynb) for more details on how to configure credentials and connect to a workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "gather": {
+     "logged": 1634852261744
+    },
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "name": "mlclient-setup",
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: C:\\Projects\\azureml-examples\\sdk\\.azureml\\config.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "credential = DefaultAzureCredential()\n",
+    "ml_client = None\n",
+    "try:\n",
+    "    ml_client = MLClient.from_config(credential)\n",
+    "except Exception as ex:\n",
+    "    print(ex)\n",
+    "    # Enter details of your AML workspace\n",
+    "    subscription_id = \"dbd697c3-ef40-488f-83e6-5ad4dfb78f9b\"\n",
```

Comment:
subscription_id = "<SUBSCRIPTION_ID>"
    resource_group = "<RESOURCE_GROUP>"
    workspace = "<AML_WORKSPACE_NAME>"

Reply:
This code is still referring to your workspace. Please update it.

Reply:
Thanks Phani.